themes/mongodb/src/js/componentCodeBlockFix.js

```diff
@@ -0,0 +1,44 @@
+function isLineNumberBlock(block) {
+    return Boolean(block.getElementsByClassName('linenos').length);
+}
+
+function hasButtonRow(block) {
+    return Boolean(block.getElementsByClassName('button-row')[0]);
+}
+
+function hasAtLeastOneButton(block) {
+    return Boolean(block.getElementsByClassName('button-row')[0].children.length);
+}
+
+function getButtonRow(block) {
+    return hasButtonRow(block)
+        ? block.getElementsByClassName('button-row')[0]
+        : null;
+}
+
+function moveButtonRowToTable(block) {
+    const br = getButtonRow(block);
+    const tdLinenos = document.createElement('td');
+    tdLinenos.id = 'linenos-button-row-filler';
```

Comment:
Should this be a class rather than an id? That is, can't there be multiple of these instances? Duplicate ids are invalid.